apps/services/endorsements/api/src/app/modules/endorsement/endorsement.service.ts

```diff
@@ -380,4 +388,119 @@ export class EndorsementService {
+    const doc = new PDFDocument()
+    const locale = 'is-IS'
+    const big = 16
+    const regular = 8
+    doc
+      .fontSize(big)
+      .text('ISLAND.IS - þetta skjal og kerfi er í vinnslu')
+      .fontSize(regular)
+      .text(
+        'þetta skjal var framkallað sjálfvirkt þann: ' +
+          new Date().toLocaleDateString(locale),
+      )
+      .moveDown()
+      .fontSize(big)
+      .text('Meðmælendalisti')
+      .fontSize(regular)
+      .text('id: ' + endorsementList.id)
+      .text('titill: ' + endorsementList.title)
+      .text('lýsing: ' + endorsementList.description)
+      .text('eigandi: ' + endorsementList.owner)
+      .text(
+        'listi opnaður: ' +
+          endorsementList.openedDate.toLocaleDateString(locale),
+      )
+      .text(
+        'listi lokaður: ' +
+          endorsementList.closedDate.toLocaleDateString(locale),
+      )
+      .text(`Alls undirskriftir: ${endorsementList.endorsements.length}`)
+      .moveDown()
+    if (endorsementList.endorsements.length) {
+      doc.fontSize(big).text('Meðmælendur').fontSize(regular)
+      for (const val of endorsementList.endorsements) {
+        doc.text(
+          val.created.toLocaleDateString(locale) + ' ' + val.meta.fullName,
+        )
+      }
+    }
+    doc.end()
+    return await getStream.buffer(doc)
+  }
+
+  async emailPDF(
+    listId: string,
+    recipientEmail: string,
+  ): Promise<{ success: boolean }> {
+    const endorsementList = await this.endorsementListModel.findOne({
+      where: { id: listId },
+      include: [
+        {
+          model: Endorsement,
+        },
+      ],
+    })
+    try {
+      const result = this.emailService.sendEmail({
+        from: {
+          name: 'TEST:Meðmælendakerfi island.is',
+          address: 'noreply@island.is',
+        },
+        to: [
+          {
+            // message can be sent to any email so recipient name in unknown
+            name: recipientEmail,
+            address: recipientEmail,
+          },
+        ],
+        subject: 'TEST:Afrit af meðmælendalista',
+        template: {
+          title: 'Afrit af meðmælendalista',
+          body: [
+            {
+              component: 'Heading',
+              context: { copy: 'Afrit af meðmælendalista' },
+            },
+            { component: 'Copy', context: { copy: 'Góðan dag.' } },
+            {
+              component: 'Copy',
+              context: {
+                copy: `... copy copy copy`,
+              },
+            },
+            {
+              component: 'Copy',
+              context: {
+                copy: `Þetta kjal er í þróun ...`,
+              },
+            },
+            {
+              component: 'Button',
+              context: {
+                copy: 'Takki',
+                href: 'http://www.island.is',
+              },
+            },
+            { component: 'Copy', context: { copy: 'Með kveðju,' } },
+            { component: 'Copy', context: { copy: 'TEST' } },
```

Comment:
This looks like test code.
Will this become visible to regular users if it were to go on prod?

Reply:
the whole app is not visible on production - so this is me getting it to dev staging and finally to production to see if the emailing functionality works on those stages

Reply:
:+1: